packages/orbit-components/src/Select/Select.stories.tsx

```diff
@@ -16,237 +17,230 @@ const objectOptions = [
   parameters: {
-    info: "Selects are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "Default setup of Select component. Check Orbit.Kiwi for more detailed guidelines.",
   },
 };
 
-export const WithCountryFlagPrefix = ({ code }) => {
-  return (
-    <Select
-      label="Select box (with prefix)"
-      options={objectOptions}
-      onChange={action("onChange")}
-      prefix={<CountryFlag code={code} />}
-    />
-  );
-};
+export const WithPrefixAndLabel: Story = {
+  render: ({ icon, ...args }) => {
+    const Icon = typeof icon === "string" && getIcon(icon);
 
-WithCountryFlagPrefix.story = {
-  name: "With CountryFlag prefix",
+    return <Select {...args} prefix={<Icon />} />;
+  },
 
-  parameters: {
-    info: "Selects are used for showing content hierarchy and are important for improving the reading experience for our users. Visit Orbit.Kiwi for more detailed guidelines.",
+  args: {
+    icon: "Airplane",
+    label: "Select box",
   },
-};
 
-WithCountryFlagPrefix.args = {
-  code: CODES.ANYWHERE,
-};
+  argTypes: {
+    icon: {
+      options: Object.keys(Icons),
+      control: {
+        type: "select",
+      },
+    },
+  },
 
-WithCountryFlagPrefix.argTypes = {
-  code: {
-    options: Object.values(CODES),
-    control: {
-      type: "select",
+  parameters: {
+    info: "Select component with prefix icon and label. Check Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onChange", "onBlur", "onFocus", "help", "error", "placeholder", "inlineLabel"],
     },
   },
 };
 
-export const WithLongLabel = ({ inlineLabel }) => {
-  return (
-    <Select
-      label="Select box (with long label)"
-      options={objectOptions}
-      onChange={action("onChange")}
-      inlineLabel={inlineLabel}
-    />
-  );
-};
+export const WithCountryFlagPrefix: Story = {
+  render: ({ code, ...args }) => {
+    const Icon = <CountryFlag code={code} />;
 
-WithLongLabel.story = {
-  name: "With long label",
+    return <Select {...args} prefix={Icon} />;
+  },
 
-  parameters: {
-    info: "Long labels truncate automatically when inline.",
+  args: {
+    code: CODES.ANYWHERE,
   },
-};
 
-WithLongLabel.args = {
-  inlineLabel: true,
-};
+  argTypes: {
+    code: {
+      options: Object.values(CODES),
+      control: {
+        type: "select",
+      },
+    },
+  },
 
-export const WithPlaceholder = ({ placeholder }) => {
-  return (
-    <Select
-      label="Select box (with placeholder)"
-      placeholder={placeholder}
-      options={objectOptions}
-      onChange={action("onChange")}
-    />
-  );
+  parameters: {
+    info: "Select component with country flag icon as a prefix. Check Orbit.Kiwi for more detailed guidelines.",
+  },
 };
 
-WithPlaceholder.story = {
-  name: "With placeholder",
+export const WithLongLabel: Story = {
+  args: {
+    label: "Select box (very long label)",
+  },
```

Comment:
Why did you remove `inlineLabel`? I think it made sense to be here - especially if you are saying in the `info` that `long label truncate automatically when inline`.

Reply:
Why did you remove `inlineLabel`? I think it made sense to be here - especially if you are saying in the `info` that `long label truncate automatically when inline`.

Reply:
I think it was there during some of the "draft" version, but it disappeared later and I didn't notice that. GP.